examples/teleport-usage/teleport_usage.py

```diff
@@ -0,0 +1,115 @@
+import json
+import datetime
+import boto3
+import time
+import os
+
+from boto3.dynamodb.conditions import Key, Attr
+
+
+# Check for AWS DynamoDB table name and region ENV, if none prompt
+TABLE_NAME = os.getenv("TABLE_NAME")
+if TABLE_NAME == None:
+    TABLE_NAME = input("DynamoDB events table name: ")
+
+aws_region = os.getenv("AWS_REGION")
+if aws_region == None:
+    aws_region = input("AWS Region for DynamoDB: ")
+
+# Check ENV for start and end times, if none, set last 30 days
+start_date = os.getenv("START_DATE")
+if start_date == None:
+    now = datetime.datetime.now()
+    start_date = now - datetime.timedelta(days=30)
+    
+end_date = os.getenv("END_DATE")
+if end_date == None:
+    end_date = now
+
+# Configure DynamoDB for query
+dynamodb = boto3.resource("dynamodb", region_name=aws_region)
+table = dynamodb.Table(TABLE_NAME)
+
+# variables for protocols
+app = 'app.session.chunk'

```

Comment:
Any reason not to use `app.session.start` instead?

If session recording is not enabled, you might not see any session chunks.

Reply:
+1

Reply:
When I was testing, I found that `app.session.start` appeared even if the app connection was not successful. While `app.session.chunk` only showed on successful launches. 

Setting to `app.session.start` will be more efficient as there are fewer events so I should revert to that. I don't think it should matter if it was successful or not.

Reply:
Really? How could you tell that the connection was not successful? I don't see any event code indicating a failed `app.session.start`.

Reply:
I happened to notice when I was testing an app which just went to chronic redirects. The audit logs would show the app session data but not the session started in the audit logs

Reply:
@zmb3 having a hard time thinking about how our event emitting works again - any idea what would cause that behavior?

Reply:
I don't think it's related to our event emitting, app access has suffered from _chronic redirects_ for ages.